docs/schema-registry.md

```diff
@@ -0,0 +1,33 @@
+# Configuration Schema Registry
+
+Although we support the use of schemas without external dependencies such as Schema Registry, it is common to use it, since it allows to
+have a control of the schemas with versions.
+
+To choose the use of Schema Registry it is necessary to do it in the following way introducing the basic properties that appear in the
+image:
+![](images/SchemaRegistrySelection.png)
+
+The properties are supported by the schema registry available to us (At the moment are Confluent Schema Registry, and Apicurio Schema Registry)
```

Comment:
I think that it's `at` (without capitalize the first letter), and I don't see the subject, maybe it would be better `Confluent Schema Registry, and Apicurio Schema Registry are available`

Reply:
In theory, when starting with a noun or a new phrase within a parenthesis, it is appropriate to begin with the first word in capital letters, unless you want remove the parenthesis, and in the other way, the subject not in the Schema Registry configuration, is located in the subject tab